app/models/vacancy.rb

```diff
@@ -278,10 +278,6 @@ def salary_types
     ]
   end
 
-  def other_contact_email(current_publisher)
-    contact_email? && contact_email != current_publisher.email
-  end
-
```

Comment:
This method needed to be renamed to `other_contact_email?` to pass `rubocop` checks. Found no calls to it. So removed it instead.